pygsti/tools/optools.py

```diff
@@ -306,9 +306,6 @@ def diamonddist(a, b, mx_basis='pp', return_x=False):
 
     # currently cvxpy is only needed for this function, so don't import until here
     import cvxpy as _cvxpy
-    old_cvxpy = bool(tuple(map(int, _cvxpy.__version__.split('.'))) < (1, 0))
-    if old_cvxpy:
-        raise RuntimeError('CVXPY 0.4 is no longer supported. Please upgrade to CVXPY 1.0 or higher.')
 
```

Comment:
Is this the official @rileyjmurray endorsed fix for this? Do we no longer need this check?

Reply:
Bump for @rileyjmurray

Reply:
This was the official endorsed fix. It's no longer needed, so I support removing.